src/index.ts

```diff
@@ -530,14 +530,13 @@ export function createTxComplex(
 export function createTx(
   from: string,
   to: string,
-  amount: string,
-  _fee: bigint,
+  amount: bigint | string,
   blockhash: string
 ): string {
-  const amountNum = Decimal.decode(amount);
+  amount = typeof amount === 'string' ? Decimal.decode(amount) : amount;
```

Comment:
Let's ensure / validate amount is bigint at this point. Needs assertion.

Reply:
It will be validated in two lines below in `sys.transferSol` method.